context/app/static/js/components/Routes/useSendWebVitals.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect } from 'react';
+import ReactGA from 'react-ga';
+import { getCLS, getFID, getLCP, getTTFB } from 'web-vitals';
+
+function sendVitalsToGA({ name, delta, id }) {
```

Comment:
```suggestion
// copy-and-paste from https://github.com/GoogleChrome/web-vitals#using-analyticsjs
function sendVitalsToGA({ name, delta, id }) {
```
credit / blame where it's due.

Reply:
Sure, I should have been more detailed with my 'from web-vitals' comments.